src/manager.vue

```diff
@@ -28,16 +37,19 @@ export default {
   methods: {
     // Public
     showToast(message, options) {
+      options = Object.assign(this.options, options || {})
```

Comment:
I think we should use here 
```
if (isObject(option)) {
   options = Object.assign({}, this.options, option)
}
```
it help dont change this.options, for this we have setOptions function.
``isObject`` we can add it in ``utils.js`` file